### ISSUE TITLE
users who dont have microsoft store

### ISSUE THREAD

**Issue by saturnw** [veler/DevToys#318]
### What's the Problem?

Good afternoon guys,

Do you have option to install this away from Ms Store, as users are not able to access Ms Store.

### Solution/Idea

stand alone installer would be great.

### Alternatives

Look forward to hearing from you.

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

**Comment by alkuzad** [comment]
https://store.rg-adguard.net/

**Comment by alkuzad** [comment]
@rlm96 winget actually uses msstore if not added explictly into packages:

`Failed to install or update Microsoft Store package because Microsoft Store client is blocked by policy`

**Comment by rlm96** [comment]
@alkuzad what about Chocolatey or Manual installation?

**Comment by Stan-C421** [comment]
A portable version should made available in my case and for my fellow colleagues. We can't use Microsoft Store due to policies, nor install on our machines.

**Comment by saturnw** [comment]
> A portable version should made available in my case and for my fellow colleagues. We can't use Microsoft Store due to policies, nor install on our machines.

We require the same.

**Comment by saturnw** [comment]
> @alkuzad what about Chocolatey or Manual installation?

Doesnt exist in choco repo.

**Comment by saturnw** [comment]
> > > @alkuzad what about Chocolatey or Manual installation?
> > 
> > 
> > Doesnt exist in choco repo.
> 
> Sorry, I have assigned the task, I will try to create it this week, I thought already existed the repo :)

please let us know would really appreacite it!

**Comment by alkuzad** [comment]
> @alkuzad what about Chocolatey or Manual installation?

I don't use choco as Scoop/WinGet are superior and automatically up-to-date.

**Comment by rlm96** [comment]
@veler is there any way we can add a .msi to the release so people can install it without compile? That way I could use the same .msi to update the Chocolatey repository

**Comment by veler** [comment]
> @veler is there any way we can add a .msi to the release so people can install it without compile? That way I could use the same .msi to update the Chocolatey repository

Unfortunately it's impossible at the moment: https://stackoverflow.com/questions/56512666/how-to-create-exe-msi-for-uwp-application

**Comment by yoshiask** [comment]
Requests for a portable version should be made in a separate issue. With some workarounds, it might be possible to create a non-Store installer, but there is no way to make a portable version.

**Comment by tmalaher-telus** [comment]
See also [issue 286](https://github.com/veler/DevToys/issues/286) 
The standalone (manual) installer is not working for me.

**Comment by veler** [comment]
Hello,
Thanks to @rlm96 , DevToys is now available on Chocolatey. This doesn't require to pass through the Microsoft Store. Would you be able to give it a try and see if the app can be installed this way on your machine, please? :)

See: https://github.com/veler/DevToys#chocolatey

Thank you !

**Comment by kevstev01** [comment]
Thanks - that worked for me (Win 10 Enterprise)

**Comment by tmalaher-telus** [comment]
See commment on issue #286 by @Atrejoe . Chocolatey is giving more details, but still failing.

**Comment by tmalaher-telus** [comment]
@veler 
Still failing. Just tried 1.0.7.0_neutral_._j80j2txgjg9dj msixbundle, and got:
`App Installer failed to install package dependencies. Ask the developer for  package. `
